src/Umbraco.Cms.Api.Management/Controllers/Server/ConfigurationServerController.cs

```diff
@@ -15,12 +17,20 @@ public class ConfigurationServerController : ServerControllerBase
     private readonly SecuritySettings _securitySettings;
     private readonly GlobalSettings _globalSettings;
     private readonly IBackOfficeExternalLoginProviders _externalLoginProviders;
+    private readonly IHostingEnvironment _hostingEnvironment;
 
-    public ConfigurationServerController(IOptions<SecuritySettings> securitySettings, IOptions<GlobalSettings> globalSettings, IBackOfficeExternalLoginProviders externalLoginProviders)
+    public ConfigurationServerController(IOptions<SecuritySettings> securitySettings, IOptions<GlobalSettings> globalSettings, IBackOfficeExternalLoginProviders externalLoginProviders, IHostingEnvironment hostingEnvironment)
```

Comment:
```suggestion
    [ActivatorUtilitiesConstructor]
    public ConfigurationServerController(IOptions<SecuritySettings> securitySettings, IOptions<GlobalSettings> globalSettings, IBackOfficeExternalLoginProviders externalLoginProviders, IHostingEnvironment hostingEnvironment)
```

I believe this attribute is necessary to tell dotnet to use this constructor - by doing that we avoid it picking the other one.  It would still work, but avoids us having to use the `StaticServiceProvider` and we keep that just to avoid a binary breaking change.